job-conf.rst

```diff
@@ -107,6 +107,18 @@ Puts a cap on the number of sites belonging to a given job that can be brozzled
 simultaneously across the cluster. Addresses the problem of a job with many
 seeds starving out other jobs.
 
+``pdfs_only``
+~~~~~~~~~~~~~~~~~~~~~
++---------+----------+-----------+
+| type    | required | default   |
++=========+==========+===========+
+| boolean | no       | ``false`` |
++---------+----------+-----------+
+Limits capture to PDFs based on MIME type. This value will only impact
+processing of outlinks within Brozzler. Fully limiting a crawl to only PDFs
+requires an additional entry in the Warcprox-Meta header ``mime-type-filters``
+key to fully block videos by MIME type.
```

Comment:
should "block videos" be "block PDFs" here?

Reply:
That clause in the sentence was redundant, so I just deleted it. Thank you!